dot/rpc/modules/payment.go

```diff
@@ -49,7 +49,6 @@ func (p *PaymentModule) QueryInfo(_ *http.Request, req *PaymentQueryInfoRequest,
 	if err != nil {
 		return err
 	}
```

Comment:
nit out of scope / unneeded delta

```suggestion
	}

```